### PR TITLE
Enable pydocstyle pre-commit hook

### PR DESCRIPTION
**Pull request by VannTen**
WIP. Fix commits welcome

### CODE REVIEW

**Comment by VannTen** [comment]
Ready for review :+1:

File: setup.py [hunk 1]
```diff
@@ -6,19 +8,18 @@
...
         # TODO: respect hashes in requirements.txt file
         res = requirements_file.readlines()
         return [req.split(" ", maxsplit=1)[0] for req in res if req]
 
 
-def get_test_requires():
-    with open("requirements-test.txt", "r") as requirements_file:
-        res = requirements_file.readlines()
-        return [req.split(" ", maxsplit=1)[0] for req in res if req]
+get_test_requires = get_install_requires
```
Comment (mayaCostantini) [inline setup.py:18]: Don't we need to get `requirements-test.txt` here instead of `requirements.txt`?

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+"""Setup configuration for adviser module."""
```
Comment (mayaCostantini) [inline setup.py:1]:
```suggestion
"""Setup configuration for storages module."""
```

File: setup.py [hunk 3]
```diff
@@ -6,19 +8,18 @@
 
 
 def get_install_requires():
+    """Get requirements for adviser module."""
```
Comment (mayaCostantini) [inline setup.py:11]:
```suggestion
    """Get requirements for storages module."""
```

File: setup.py [hunk 4]
```diff
@@ -6,19 +8,18 @@
...
 
-def get_test_requires():
-    with open("requirements-test.txt", "r") as requirements_file:
-        res = requirements_file.readlines()
-        return [req.split(" ", maxsplit=1)[0] for req in res if req]
+get_test_requires = get_install_requires
 
 
 def get_version():
+    """Get current version of adviser module."""
```
Comment (mayaCostantini) [inline setup.py:22]:
```suggestion
    """Get current version of storages module."""
```

**Review by mayaCostantini — COMMENTED** [review]
lgtm, just a few details :+1:

**Comment by VannTen** [comment]
On Tue, Jul 26, 2022 at 03:25:44AM -0700, Maya Costantini wrote:
> @mayaCostantini commented on this pull request.
> 
> lgtm, just a few details :+1: 
> 
> >      with open("requirements.txt", "r") as requirements_file:
>          # TODO: respect hashes in requirements.txt file
>          res = requirements_file.readlines()
>          return [req.split(" ", maxsplit=1)[0] for req in res if req]
>  
>  
> -def get_test_requires():
> -    with open("requirements-test.txt", "r") as requirements_file:
> -        res = requirements_file.readlines()
> -        return [req.split(" ", maxsplit=1)[0] for req in res if req]
> +get_test_requires = get_install_requires
> 
> Don't we need to get `requirements-test.txt` here instead of `requirements.txt`?


Oh that's embarrassing. Code-climate was yelling that the two code
blocks were identical and my eyes could not make the diff.

> 
> > @@ -1,3 +1,5 @@
> +"""Setup configuration for adviser module."""
> 
> ```suggestion
> """Setup configuration for storages module."""
> ```
> 
> > @@ -6,19 +8,18 @@
>  
>  
>  def get_install_requires():
> +    """Get requirements for adviser module."""
> 
> ```suggestion
>     """Get requirements for storages module."""
> ```
> 
> >  
>  
>  def get_version():
> +    """Get current version of adviser module."""
> 
> ```suggestion
>     """Get current version of storages module."""
> ```
> 

ACK, I'll fix all of that.

**Comment by mayaCostantini** [comment]
Thanks @VannTen :smile:

**Review by harshad16 — APPROVED** [review]
/lgtm

one note would be squash the commits to 1 . 
thanks :100:

**Comment by VannTen** [comment]
Agree for some squashing. I kept 3 distinct commits though, to separate
docstrings modifications from refactors.

**Comment by VannTen** [comment]
I had forgotten the `adviser` -> `storages` fix I don't know why.
This should be ready now :+1:

**Review by mayaCostantini — APPROVED** [review]
lgtm, thanks for the changes :100:

**Pull request merged**